src/Model/Settings.php

```diff
@@ -209,11 +210,12 @@ private function meta_global_fields(): array {
 	 */
 	private function meta_social_fields(): array {
 		return [
-			'facebookPageUrl'   => ! empty( $this->data['titles']['social_url_facebook'] ) ? $this->data['titles']['social_url_facebook'] : null,
-			'facebookAuthorUrl' => ! empty( $this->data['titles']['facebook_author_urls'] ) ? $this->data['titles']['facebook_author_urls'] : null,
-			'facebookAdminId'   => ! empty( $this->data['titles']['facebook_admin_id'] ) ? $this->data['titles']['facebook_admin_id'] : null,
-			'facebookAppId'     => ! empty( $this->data['titles']['facebook_app_id'] ) ? $this->data['titles']['facebook_app_id'] : null,
-			'twitterAuthorName' => ! empty( $this->data['titles']['twitter_author_names'] ) ? $this->data['titles']['twitter_author_names'] : null,
+			'facebookPageUrl'    => ! empty( $this->data['titles']['social_url_facebook'] ) ? $this->data['titles']['social_url_facebook'] : null,
+			'facebookAuthorUrl'  => ! empty( $this->data['titles']['facebook_author_urls'] ) ? $this->data['titles']['facebook_author_urls'] : null,
+			'facebookAdminId'    => ! empty( $this->data['titles']['facebook_admin_id'] ) ? $this->data['titles']['facebook_admin_id'] : null,
+			'facebookAppId'      => ! empty( $this->data['titles']['facebook_app_id'] ) ? $this->data['titles']['facebook_app_id'] : null,
+			'twitterAuthorName'  => ! empty( $this->data['titles']['twitter_author_names'] ) ? $this->data['titles']['twitter_author_names'] : null,
+			'additionalProfiles' => ! empty( $this->data['titles']['social_additional_profiles'] ) ? Arr::from_string( $this->data['titles']['social_additional_profiles'], PHP_EOL ) : null,
```

Comment:
RankMath stores this using site-specific new lines (i.e. `\n` or `\r\n` ). by using `Arr::from_string()` we explore it the same way RM does internally in `RankMath\Schema\JsonLd::get_social_profiles()`